bin/prepare-release.js

```diff
@@ -72,6 +72,7 @@ async function prepareReleaseNotes() {
   const options = program.opts()
   console.log('Script running with following options: ', JSON.stringify(options))
   const [owner, repo] = options.repo.split('/')
+  const [, , baseBranch] = options.branch.split('/')
```

Comment:
This isn't going to work for simple names or for differently styled refs. For example:

```js
const [, , foo] = 'main'.split('/')
// foo => undefined
```

My understanding of `--branch` is that it is the branch to generate release notes from. It is not necessarily the branch to target with the resulting PR.

Reply:
True--the intention was to collect the release notes from some branch, most frequently `main`. The plugin repo, though, uses `master`, which in GHA was expressed as `refs/heads/master`.  In any case, I'm closing this because I've renamed the main branch.